docs/CONFIGURATION.md

```diff
@@ -103,12 +103,18 @@ graph LR;
 
 #### Service Account configuration for EKS Clusters
 
-EKS allows [using Kubernetes service accounts to authenticate requests to S3](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html).
-To set this up follow these steps:
+EKS allows using Kubernetes service accounts to authenticate requests to S3 using [IAM Roles for Service Accounts (IRSA)](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html).
+This is supported for both driver-level and pod-level identity.
```

Comment:
Should driver-level and pod-level here link to corresponding sections?

Reply:
Updated with links (in both occurrences).